pkg/operator/edgefs/cluster/target/config/config.go

```diff
@@ -107,6 +119,13 @@ func ToStoreConfig(config map[string]string) StoreConfig {
 			storeConfig.UseAllSSD = convertToBoolIgnoreErr(v)
 		case RtrdPlevelOverrideKey:
 			storeConfig.RtrdPLevelOverride = convertToIntIgnoreErr(v)
+		case SyncKey:
+			value := convertToIntIgnoreErr(v)
+			if value >= 0 && value <= 3 {
+				storeConfig.Sync = value
+			} else {
+				logger.Warningf("Incorrect 'sync' value %d ignored", value)
```

Comment:
Good to see these added, though not all other key/value pairs in this `case` statement have them know, is that wanted?